support-frontend/assets/components/contributionsReminder/contributionsReminder.jsx

```diff
@@ -31,6 +36,28 @@ const mapStateToProps = state => ({
 
 // ----- Render ----- //
 class ContributionsReminder extends Component<PropTypes, StateTypes> {
+  reminderDates: Array<ReminderDate>;
```

Comment:
looks like this can just be defined as a const outside of the class?

Reply:
agreed 👍